webcomponents/tool/src/main/frontend/js/sakai-permissions.js

```diff
@@ -65,6 +65,16 @@ class SakaiPermissions extends SakaiElement {
   get tool() { return this._tool; }
 
   updated(changedProperties) {
+    let inputs = this.renderRoot.querySelectorAll("input[type='checkbox']");
+    inputs.forEach(elem => { 
+      let role = elem.getAttribute('data-role');
```

Comment:
blank line here please, it makes js easier to read, imo, if you leave a blank line at the start of any multi line function body, even lambdas.